game/scripts/vscripts/items/drums_of_endurance.lua

```diff
@@ -0,0 +1,262 @@
+LinkLuaModifier( "modifier_intrinsic_multiplexer", "modifiers/modifier_intrinsic_multiplexer.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa_swiftness_aura", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa_swiftness_aura_effect", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_drums_of_endurance_oaa_active", "items/drums_of_endurance.lua", LUA_MODIFIER_MOTION_NONE )
+
+item_drums_of_endurance_oaa = class(ItemBaseClass)
+
+function item_drums_of_endurance_oaa:GetIntrinsicModifierName()
+  return "modifier_intrinsic_multiplexer"
+end
+
+function item_drums_of_endurance_oaa:GetIntrinsicModifierNames()
+  return {
+    "modifier_item_drums_of_endurance_oaa",
+    "modifier_item_drums_of_endurance_oaa_swiftness_aura"
```

Comment:
I realised that this probably does not have good interactions when you have multiple Drums. I think you should just make it a single modifier after all.

Reply:
Merge the functionality, then replace the intrinsic multiplexer with the single merged modifier. You could choose either name to keep, but of course it makes more sense to use `modifier_item_drums_of_endurance_oaa` for the name.

Reply:
Got it